python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -232,6 +238,57 @@ def __new__(
             has_uninstalled_deps=has_uninstalled_deps,
             manifest_preparer=manifest_preparer,
         )
-        if manifest_preparer:
-            manifest_preparer.on_load(val)
-        return val
+
+    @public
+    def ensure_prepared(self) -> "DbtProject":
+        """Ensure that the preparation process is executed for a dbt project and
```

Comment:
To be as direct as possible, I would lead with something like "Ensure that the dbt manifest file and dbt deps are available locally and up-to-date."

I think it's also important to:
- Mention what circumstances this will error under
- Mention what the environment variable is that controls its behavior

Reply:
@sryza I updated the docstring to explain the current default behavior, ensured by `DagsterDbtManifestPreparer`. 

For the environment variables, are we comfortable discussing them in the `DagsterDbtManifestPreparer`'s docstring? It seems very specific to that implementation.